src/transforms/operationTransforms.ts

```diff
@@ -133,7 +133,13 @@ export function transformOperationSpec(
   return operationSpecDetails;
 }
 
-export function extractHttpDetails({ path, method }: OperationRequestDetails) {
+export function extractHttpDetails({
+  path,
+  method
+}: OperationRequestDetails): {
+  path: string;
+  httpMethod: HttpMethods;
+} {
```

Comment:
for some reason the compiler did not like leaving the return type implicit so I had to add it.